### PR TITLE
JSON for Modern C++ 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by nlohmann**
JSON for Modern C++ is a single-header library for C++11. Its goal is to introduce JSON as first-class data type. JSON is implemented as C++ container and plays well with other STL containers, algorithms, or other aspects of C++.

### CODE REVIEW

File: Formula/nlohmann_json.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
+class NlohmannJson < Formula
+  desc "JSON for Modern C++"
+  homepage "https://github.com/nlohmann/json"
+  url "https://github.com/nlohmann/json/archive/v1.1.0.tar.gz"
+  sha256 "ee3825841e6d6915428caf2cea53927c3e4d56315a23ee7d1a64bfe1c19a656f"
+  head "https://github.com/nlohmann/json.git"
+
+  def install
+    include.install "src/json.hpp"
+    ohai "to use the library, please set your include path accordingly:"
```
Comment (DomT4) [inline Formula/nlohmann_json.rb]: Please don't `ohai` here.

File: Formula/nlohmann_json.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
+class NlohmannJson < Formula
+  desc "JSON for Modern C++"
+  homepage "https://github.com/nlohmann/json"
+  url "https://github.com/nlohmann/json/archive/v1.1.0.tar.gz"
+  sha256 "ee3825841e6d6915428caf2cea53927c3e4d56315a23ee7d1a64bfe1c19a656f"
+  head "https://github.com/nlohmann/json.git"
+
+  def install
+    include.install "src/json.hpp"
```
Comment (DomT4) [inline Formula/nlohmann_json.rb]: Various maintainers have feelings on single-header formulae, so I'll open this one up for comments.

Comment (mikemcquaid) [inline Formula/nlohmann_json.rb]: Yeh, personally 👎 on these formulae; feels like a better fit for projects to vendor them themselves.

File: Formula/nlohmann_json.rb (hunk 1 above)
Comment (nlohmann) [inline Formula/nlohmann_json.rb]: Should the message be removed altogether our just be displayed with a command different than `ohai`?

**Comment by dunn** [comment]
Passing on this, but thank you for the submission! ❤️

**Pull request closed without merging**